src/StoreApi/Routes/Checkout.php

```diff
@@ -382,6 +382,14 @@ private function create_or_update_draft_order( \WP_REST_Request $request ) {
 				$e->getCode()
 			);
 		}
+
+		/**
+		 * Fire action after draft order creation/update.
+		 * Functions hooking into this can throw a \RouteException to force the checkout block into an error state that prevents access to its contents.
+		 *
+		 * @param \WC_Order $order Order object.
+		 */
+		do_action( 'wooocommerce_blocks_draft_order_updated', $this->order );
```

Comment:
Can you explain why this is needed specifically? Is the hook above not useful?

If we do add a new hook it needs to be added with `_experimental` prefix to validate its usefulness before committing to it.

Reply:
Thanks Mike! 

This should be covered [here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5874#issuecomment-1042955544).